frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  const error = MultipleErrorWrapper(
```

Comment:
We could memoize this so that we're not running through this logic on every render.
```
  const error = useMemo(
    () =>
      MultipleErrorWrapper(
        `Unable to load approval page`,
        {
          isApprovalsError,
        },
        ErrorWrapper
      ),
    [isApprovalsError]
  )
```